site_scons/site_tools/go_builder.py

```diff
@@ -63,21 +77,19 @@ def _check_go_version(context):
 
         context.Display(f'Checking {env.d_go_bin} version... ')
         cmd_rc = subprocess.run([env.d_go_bin, 'version'], check=True, stdout=subprocess.PIPE)
-        out = cmd_rc.stdout.decode('utf-8')
+        out = cmd_rc.stdout.decode('utf-8').strip()
         if len(out.split(' ')) < 3:
             context.Result(f'failed to get version from "{out}"')
             return 0
```

Comment:
```suggestion
```
I would just remove this, as it's not needed anymore with the rest of the changes.